module/VuFind/src/VuFind/RecordDriver/SolrDefault.php

```diff
@@ -133,6 +143,13 @@ public function __construct($mainConfig = null, $recordConfig = null,
         $this->containerLinking
             = !isset($mainConfig->Hierarchy->simpleContainerLinks)
             ? false : $mainConfig->Hierarchy->simpleContainerLinks;
+
+        // Record versions display limit
+        if (!empty($searchSettings->General->record_versions_limit)) {
```

Comment:
I think it's probably worth putting this in searches.ini as a commented-out setting, just because the .ini files serve as our configuration documentation, and this would be the only undocumented setting, as far as I know.

Reply:
Actually, I'm not sure the limit is useful at all since the count is currently fixed in the versions template. The limit can be added later if we ever add a selectable number of records.

Reply:
I have no objection to leaving it non-configurable; I'd just like to avoid having a "magic number" if we can use a property or constant, just to make it more clear what the number means and to make it easier to change/customize if the need arises.

Reply:
I see you've just removed it entirely, which also nicely solves the problem. :-)